12-Principles-Of-Agile-Manifesto.md

```diff
@@ -0,0 +1,29 @@
+# 12 Principles Behind The Agile Manifesto
+
+### We follow these principles:
+
+1. Our highest priority is to satisfy the customer through early and continuous delivery of valuable software.
+
+2. Welcome changing requirements, even late in development. Agile processes harness change for the customer's competitive advantage.
+
+3. Deliver working software frequently, from a couple of weeks to a couple of months, with a preference to the shorter timescale.
+
+4. Business people and developers must work together daily throughout the project.
+
+5. Build projects around motivated individuals. Give them the environment and support they need, and trust them to get the job done.
+
+6. The most efficient and effective method of conveying information to and within a development team is face-to-face conversation.
+
+7. Working software is the primary measure of progress.
+
+8. Agile processes promote sustainable development. The sponsors, developers, and users should be able 
+to maintain a constant pace indefinitely.
+
+9. Continuous attention to technical excellence and good design enhances agility.
+
+10. Simplicity -- * the art of maximizing the amount of work not done * -- is essential.
+
+11. The best architectures, requirements, and designs emerge from self-organizing teams.
```

Comment:
I'm not sure about this one. Are you sure this change should be made? Also this was directly copied, it is open for anyone to use, directly from the agile manifesto website.